website/docs/Conditions.md

```diff
@@ -0,0 +1,166 @@
+---
+slug: Conditions.md
+sidebar_position: 20
+---
+
+# Conditions
+Several types of access conditions can be defined:
+- EVM - on chain state, eg NFT ownership, ETH balance, tx status, contract function call
+- RPC - ethereum RPC calls as defined in the [Official API](https://ethereum.org/en/developers/docs/apis/json-rpc/#json-rpc-methods)
+- Timelock - time based conditions, eg Block Time, Block Height, UTC Time
+
+## EVM Conditions
+Here is an example EVM condition for ownership of a specific ERC721 token (NFT):
+
+```js
+const ERC721Conditions = {
+    contractAddress: "0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D",
+    standardContractType: "ERC721",
+    chain:  "ethereum",
+    method: "ownerOf",
+    parameters: [
+      "5954"
+    ],
+    returnValueTest: {
+      comparator: "=",
```

Comment:
```suggestion
      comparator: "==",
```

I suspect this was taken from here - https://github.com/nucypher/nucypher/blob/tdec/tests/data/test_conditions.json#L51 - @KPrasch this test condition should be updated - doesn't seem to fail testing though, maybe unused for now.